x/ecocredit/server/core/keeper_test.go

```diff
@@ -36,6 +36,7 @@ type baseSuite struct {
 	addr         sdk.AccAddress
 	bankKeeper   *mocks.MockBankKeeper
 	paramsKeeper *mocks.MockParamKeeper
+	moduleAddress sdk.AccAddress
```

Comment:
does this need to be a baseSuite field? its never used outside of keeper instantiation

Reply:
https://github.com/regen-network/regen-ledger/pull/1086/commits/86c6b6de1c0fe529a3a2537f091ba878564692fc